lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js

```diff
@@ -106,6 +106,13 @@ define(function(require) {
       event.preventDefault();
       Modal.close(false);
     });
+
+    // Close the modal when a user wants to change their photo.
+    this.$cropModal.find(".-change").click(function(event) {
+      event.preventDefault();
+      window.location.href = "#prove";
```

Comment:
Problem is we're having some issues needing to clear out the file-input file, so we're temporarily doing a page refresh to reset until we can do a proper fix.